ext/mysqli/mysqli.stub.php

```diff
@@ -256,10 +256,10 @@ public function set_opt(int $option, $value) {}
      */
     public function ssl_set(
         string $key,
-        string $cert,
+        string $certificate,
         string $certificate_authority,
         string $certificate_authority_path,
```

Comment:
```suggestion
        string $ca_path,
```
to be consistent with that.